superset-frontend/src/views/CRUD/data/components/SyntaxHighlighterCopy/index.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { styled, t } from '@superset-ui/core';
+import { SyntaxHighlighterProps } from 'react-syntax-highlighter';
+import sql from 'react-syntax-highlighter/dist/cjs/languages/hljs/sql';
+import github from 'react-syntax-highlighter/dist/cjs/styles/hljs/github';
+import SyntaxHighlighter from 'react-syntax-highlighter/dist/cjs/light';
+import { ToastProps } from 'src/messageToasts/enhancers/withToasts';
+import Icon from 'src/components/Icon';
+
+SyntaxHighlighter.registerLanguage('sql', sql);
+
+const SyntaxHighlighterWrapper = styled.div`
+  margin-top: -24px;
+  &:hover {
+    svg {
+      visibility: visible;
+    }
+  }
+  svg {
+    position: relative;
+    top: 40px;
+    left: 512px;
+    visibility: hidden;
+    margin: -4px;
```

Comment:
small nit, but do you need the negative margin if you have the top and left relative positions?

Reply:
That negative margin gets rid of the space that would be occupied by the element if it were not relative positioned. If we were using absolute positioning we wouldn't need it.